cfgov/v1/wagtail_hooks.py

```diff
@@ -19,6 +19,11 @@
 logger = logging.getLogger(__name__)
 
 
+@hooks.register('before_delete_page')
+def raise_delete_error(request, page):
+    raise Exception('Deletion via POST is disabled')
```

Comment:
With a 403 error the only output is "403 Forbidden" and it doesn't include the error message -- I don't have a strong opinion, but strictly speaking this isn't really a matter of permissions right?  We just don't want to allow anyone to be able to do it through the UI?